src/protocol.h

```diff
@@ -177,26 +193,37 @@ void CreateReqChannelLevelListMes();
                             const int               iRecCounter,
                             const int               iRecID );
 
-    void ParseConnectionLessMessageBody ( const CVector<uint8_t>& vecbyMesBodyData,
-                                          const int               iRecID,
-                                          const CHostAddress&     InetAddr );
+    void
+    ParseConnectionLessMessageBody ( const CVector<uint8_t>& vecbyMesBodyData,
+                                     const int               iRecID,
+                                     const CHostAddress&     InetAddr );
 
     static bool IsConnectionLessMessageID ( const int iID )
-        { return ( iID >= 1000 ) && ( iID < 2000 ); }
+    {
+        return ( iID >= 1000 ) && ( iID < 2000 );
+    }
 
     // this function is public because we need it in the test bench
-    void CreateAndImmSendAcknMess ( const int& iID,
-                                    const int& iCnt );
+    void CreateAndImmSendAcknMess ( const int& iID, const int& iCnt );
 
-protected:
+  protected:
     class CSendMessage
     {
-    public:
-        CSendMessage() : vecMessage ( 0 ), iID ( PROTMESSID_ILLEGAL ),
-            iCnt ( 0 ) {}
-        CSendMessage ( const CVector<uint8_t>& nMess, const int iNCnt,
-            const int iNID ) : vecMessage ( nMess ), iID ( iNID ),
-            iCnt ( iNCnt ) {}
+      public:
+        CSendMessage() :
```

Comment:
Mentioned elsewhere but the original here is correct and should be supported.

Reply:
clang-format will keep it in one line when the limit is not exceeded, otherwise each parameter gets its own line with the settings we have now:

> BinPackParameters: 
> ```
>  If false, a function declaration's or function definition's parameters will either all be on the same line or will have one line each.
> ```

Reply:
didn't spot what this was about at first, now fixed by `AccessModifierOffset: -4`